### PR TITLE
Adding twist to ackermann code

### PR DESCRIPTION
**Pull request by AmyPhung**

### CODE REVIEW

File: scripts/ConvertToAckermann.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        ack_msg = AckermannDrive()
+        ack_msg.speed = vel_scale(linear_vel)
+        ack_msg.steering_angle = angle_scale(angular_vel)
+
+        return ack_msg
+
+    def run(self):
+        # Takes no args, executes timed loop for node
+        while (self.twist_data == None) and not rospy.is_shutdown():
+            print('MSG: No twist data published')
```
Comment (concavegit) [inline scripts/ConvertToAckermann.py]: Use `rospy.loginfo("...")`

Comment (concavegit) [reply scripts/ConvertToAckermann.py]: It is easier and clearer to  merge the two while loops by deleting the one above and inserting this in the other while loop

```python
if self.twist_data == None:
    rospy.loginfo("...")
    continue

File: scripts/ConvertToAckermann.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+"""
+Subscribes to a twist message and converts it to an ackermann message.
+Subscribes to /twist_vel
+Publishes to /cmd_vel
+
+Assumes all inputs are normalized between -1 and 1
+
+@edited: 9/28/18
```
Comment (concavegit) [inline scripts/ConvertToAckermann.py]: Make that up-to-date. Personally, the year should be all 4 letters, though there is an argument for git taking care of date annotations for you.

File: scripts/ConvertToAckermann.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+class ConvertToAckermann():
+    def __init__(self):
+        # Define variables
+        self.twist_data = None
+
+        # Define ROS constructs
+        rospy.init_node("convert_to_ackermann")
+        self.twist_sub = rospy.Subscriber("/twist_vel", Twist, self.twist_cb)
+        self.ack_pub = rospy.Publisher("/drive", AckermannDrive, queue_size=10)
```
Comment (concavegit) [inline scripts/ConvertToAckermann.py]: If we are doing this for control, `queue_size=1` so that we are always using the last command.

**Comment by ConnorNovak** [comment]
Twist to Ackermann node added with #90

**Pull request closed without merging**